Copy turn order clarification from FAQ (#580)
Heegu-sama/Homm3BG: sections/round_structure.tex

\vspace*{\fill}

> \includegraphics[width=1.6\linewidth]{\art/minotaur.png}

Reviewer: Please move it slightly to the right.

Author: how?

Reviewer: Use `\hspace`, see Medusa in Scenario End.

Author: Done

\vspace*{\fill}

\note{9}{

>   During the Alliance game, you play in teams, which activate alternately until all players have taken their turn.

Reviewer: Why note and not regular text?

Author: Because that's an unusual rule which doesn't apply to most games, but only in certain conditions?

That's called an exception:

![Выделение_582](https://github.com/Heegu-sama/Homm3BG/assets/325092/a2f7bc5a-e5da-4070-af60-65024be91a05)

And... I've just noticed two links which should probably stay hyperlinks instead of pagelink :D

Reviewer: You mean those on the intro page?

Author: Yeah, it is quite long

Reviewer 2: Two ideas for shorter versions

> In the Alliance game, teams take turns alternately until all players have played. When it's your team's turn, you can choose who goes first if no one from your team has played that round, allowing you to swap players each turn.

> In the Alliance game, teams take turns alternately. Each round, you can choose who goes first within your team if no one has played yet.

Reviewer: +1 for the latter

>   At the start of your team's first turn of the Round, you can choose who of your team goes first and who last in that Round.

Reviewer: ```suggestion
  At the start of your team's first turn of the Round, you can choose which of the team members goes first in that Round.
```

Reviewer 2: +1 for the suggestion, the "last" part is kind of clear if you specify the one going first and there are two players on a team.

Reviewer 2: Czech translation for that
> Ve hře Alliance se týmy po každém tahu střídají. Na začátku prvního tahu vašeho týmu v daném kole si můžete vybrat, který z členů týmu půjde v tomto kole první.

Reviewer: ```suggestion
  At the start of your team's first turn of the Round, choose who goes first and last in that Round.
```
Alternative version if we want to keep the "last" word.
